client/checkout/classic/style.scss

```diff
@@ -39,53 +39,56 @@
 		max-height: 24px !important;
 	}
 
-	li.payment_method_woocommerce_payments {
-		display: grid;
-		grid-template-columns: 0fr 0fr 1fr;
-		grid-template-rows: max-content;
-
-		> input[name='payment_method'] {
-			align-self: center;
-		}
-		> label {
-			grid-column: 3;
+	&.wc_payment_methods,
+	&.woocommerce-PaymentMethods {
```

Comment:
The diff is a bit messy, but this is the only new selectors that were added, to prevent these styles to be applied in the change payment method form.